### DISCUSSION TITLE
Sort/ Order env variables for an enviornment

### DISCUSSION THREAD

**Discussion by ighoshsubho** [hoppscotch/hoppscotch#3802] [Ideas]
While getting the global env, we can return it after sorting from the liveEnvs function from "my" and "teams" enviornments. Files - 
1. https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-common/src/components/environments/teams/Details.vue
2. https://github.com/hoppscotch/hoppscotch/blob/main/packages/hoppscotch-common/src/components/environments/my/Details.vue

**Comment by ighoshsubho** [comment]
@liyasthomas does this solution sound good?

**Comment by AntoninHuaut** [comment]
This feature would be useful 😊
